apps/studio/components/interfaces/Auth/MfaAuthSettingsForm/MfaAuthSettingsForm.tsx

```diff
@@ -458,6 +475,30 @@ const MfaAuthSettingsForm = () => {
           </form>
         </Form_Shadcn_>
       </ScaffoldSection>
+      <ConfirmationModal
+        visible={isConfirmationModalVisible}
+        title="Confirm SMS MFA"
+        confirmLabel="Confirm and save"
+        onCancel={() => setIsConfirmationModalVisible(false)}
+        onConfirm={() => {
+          setIsConfirmationModalVisible(false)
+          phoneForm.handleSubmit(onSubmitPhoneForm)()
+        }}
+        variant="warning"
+      >
+        <Alert_Shadcn_ variant="warning">
+          <WarningIcon />
+          <AlertTitle_Shadcn_>
+            Enabling advanced MFA with phone will result in an additional charge of{' '}
```

Comment:
I know that you simply reused the existing text. Might make sense to use "SMS MFA" instead of "advanced MFA". Furhter down and as the headline of the form we use "SMS MFA".